CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 ### Modules
 #### Issue Tracker
 - Readability of comments and history was improved. (PR #6138)
+#### Candidate Parameters
+- Consent grouping added to SQL schema and Consent Status tab UI (PR #6042, PR #6044)
```

Comment:
```suggestion
- Consents may now be grouped UI of consent tab (PR #6042, PR #6044)
```

I don't think we need to separately tell them that the UI and SQL was done, just that the feature is implemented..